src/odemis/gui/util/test/img_test.py

```diff
@@ -1083,12 +1092,18 @@ def setUp(self):
         sem_stream_pj.mpp.value = 1e-6
 
         self.streams = [fluo_stream_pj, sem_stream_pj]
-        self.min_res = (623, 432)
+        self.image_ratio = [[] for _ in self.streams]
+        self.image_ratio[0] = fluo_stream_resolution[0] / fluo_stream_resolution[1]
+        self.image_ratio[1] = SEM_stream_resolution[0] / SEM_stream_resolution[1]
```

Comment:
My notation is indeed not that nice/readable. But I do not think the proposed solution is perfect, @tepals also had problems understanding that one. (She was thinking/hoping there would be a third and best solution?)

Maybe hard coding is just the nicest and easiest, since it is a test case and both notations implicitly do that already that is maybe not that bad?

Proposed solution:
`self.image_ration = [ [ fluo_stream_resolution[0] / fluo_stream_resolution[1]  ], [ SEM_stream_resolution[0] / SEM_stream_resolution[1] ] ]`

@pieleric what do you think about that?

Reply:
Yes, that also looks good to me 🙂